secret_number.js

```diff
@@ -1,4 +1,11 @@
 'use strict';
-module.exports = function() {
-  
+module.exports = function () {
+  var randomNumber = Math.random()*1000001;
```

Comment:
So `Math.random()` will give you a floating point number between 0 and 0.999999999999. If you times that by 1000001, you'll get at the highest value something akin to 1000000.998999999. Can you think of something to limit it to just 1000000?

Reply:
Also use const here.